doc/manual/manual.tex

```diff
@@ -5303,6 +5303,135 @@ \subsubsection{Simple convection in a spherical 3d shell}
 reality.
 
 
+\subsubsection{3D convection with an Earth-like initial condition}
+\label{sec:cookbooks-S20RTS}
+\textit{This section was contributed by Jacqueline Austermann}
+
+For any model run with \aspect{} we have to choose an initial condition for the 
+temperature field. If we want to model convection in the Earth's mantle we want 
+to choose an initial temperature distribution that captures the Earth's buoyancy 
+structure. In this cookbook we present how to use temperature perturbations 
+based on the shear wave velocity model S20RTS (\cite{S20RTS}) to 
+initialize a mantle convection calculation.
+
+\paragraph{The input shear wave model.}
+
+The current version of \aspect{} can read in the shear wave velocity models 
+S20RTS (\cite{S20RTS}) and S40RTS (\cite{S40RTS}), which are located 
+in \texttt{data/initial-conditions/S40RTS/}. Those models provide 
+spherical harmonic coefficients up do degree 20 and 40, respectively, for 21 
+depth layers. The interpolation with depth is done through a cubic spline 
+interpolation. The input files S20RTS.sph and S40RTS.sph were
+downloaded from \url{http://www.earth.lsa.umich.edu/~jritsema/Research.html} 
+and have the following format (this example is S20RTS): \\
+
+20 111111111111111111111  24 000111111111111111111111 \\
```

Comment:
In essence you have a code sample here. Can you mark it up with `lstlisting` as in other places of the code? Then you also won't need the backslashes at the ends of lines, and it should be with fixed width fonts so that it is aligned neatly.